src/modules/item/utils.ts

```diff
@@ -458,14 +458,14 @@ export function isWearableSynced(item: Item, entity: Entity) {
   return true
 }
 
-export function isEmoteSynced(item: Item, entity: Entity) {
+export function isEmoteSynced(item: Item, entity: Entity, isEmotesFeatureFlagOn: boolean) {
   if (item.type !== ItemType.EMOTE) {
     throw new Error('Item must be EMOTE')
   }
 
   // check if metadata has the new schema from ADR 74
   const isADR74 = 'emoteDataADR74' in entity.metadata
-  if (!isADR74) {
+  if (!isADR74 && isEmotesFeatureFlagOn) {
```

Comment:
Is there a way to get this boolean here without depending on an argument? I see it comes from another selector but looking at the code it seems it'd be nice to keep it locally to this file right?

Reply:
Yeah, I thought the same thing but I didn't find another way since it comes from the state, I believe it must be passed as parameter :(